config/events/config.go

```diff
@@ -0,0 +1,43 @@
+package events
+
+import (
+	"github.com/upbound/upjet/pkg/config"
+)
+
+// Configure adds configurations for events group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_cloudwatch_event_permission", func(r *config.Resource) {
+		r.References["event_bus_name"] = config.Reference{
+			Type:              "github.com/upbound/provider-aws/apis/events/v1beta1.Bus",
+			RefFieldName:      "EventBusNameRefs",
+			SelectorFieldName: "EventBusNameSelector",
+		}
+		r.UseAsync = true
```

Comment:
nit: Async is the [default](https://github.com/upbound/upjet/blob/2bd418a5521b5a4a575c0fae7185bd7f60a06560/pkg/config/common.go#L71) mode in upjet now:
```suggestion
```